### PR TITLE
Add bubble tests

### PR DESCRIPTION
**Pull request by akshaysridhar**
Adds tests and comments to the 3D RTB problem. Current time-to-solution for the 700s 3D test case might not be feasible - so test times are truncated.

### CODE REVIEW

**Comment by bischtob** [comment]
whoops

File: src/Models/Nonhydrostatic3DModels/equations_thermodynamics.jl [hunk 1]
```diff
@@ -53,7 +53,7 @@ end
     @. dρθ = -κ₄ * hwdiv(ρ * hgrad(χe))
 
     # advection 
-    @. dρθ -= hdiv(uvw * (ρθ + p))
+    @. dρθ -= hdiv(uvw * (ρθ))
```
Comment (bischtob) [inline src/Models/Nonhydrostatic3DModels/equations_thermodynamics.jl:56]: Good catch

**Review by bischtob — COMMENTED** [review]
I think this is basically ready. You can get rid of the :integration tests. We just do :regression and :validation now. Other than that it should just be imports at the top.

File: test/test_cases/run_3d_rising_bubble.jl [hunk 2]
```diff
@@ -1,22 +1,22 @@
-using CLIMAParameters
-using ClimaAtmos.Utils.InitialConditions: init_3d_rising_bubble
-using ClimaAtmos.Domains
-using ClimaAtmos.BoundaryConditions
-using ClimaAtmos.Models.Nonhydrostatic3DModels
-using ClimaAtmos.Simulations
+include("initial_conditions/dry_rising_bubble_3d.jl")
```
Comment (bischtob) [inline test/test_cases/run_3d_rising_bubble.jl:1]: Imports are different now

File: test/test_cases/run_3d_rising_bubble.jl [hunk 3]
```diff
@@ -35,32 +35,196 @@ function run_3d_rising_bubble(
...
+        domain = domain,
+        boundary_conditions = nothing,
+        thermodynamics = PotentialTemperature(),
+        parameters = params,
+        hyperdiffusivity = FT(100),
+    )
 
     # execute differently depending on testing mode
-    if test_mode == :regression
+    if mode == :integration
```
Comment (bischtob) [inline test/test_cases/run_3d_rising_bubble.jl:47]: Don't need integration. Regression+ validation is fine.

File: test/test_cases/run_3d_rising_bubble.jl [hunk 4]
```diff
@@ -35,32 +35,196 @@ function run_3d_rising_bubble(
...
+            # the addition of new model components these tests can be re-run to 
+            # ensure solution stability is unaffected, and that deviations from
+            # current variable bounds is satisfactory. 
+            #
+            # For the RTB test, the typical test time is 700s. 
+            # TODO: Fix when time-to-solution is reasonable. 
+            "Regression: Potential Temperature Model"
+            simulation =
+                Simulation(model_pottemp, stepper, dt = dt, tspan = (0.0, 1.0))
+            @unpack ρ, uh, w, ρθ = init_dry_rising_bubble_3d(FT, params, :ρθ)
```
Comment (bischtob) [inline test/test_cases/run_3d_rising_bubble.jl:94]: function name has changed

**Review by bischtob — CHANGES_REQUESTED** [review]
After this, we are done :).

File: test/test_cases/run_3d_rising_bubble.jl (hunk 3 above)
Comment (akshaysridhar) [reply test/test_cases/run_3d_rising_bubble.jl]: Noted and resolved.

File: test/test_cases/run_3d_rising_bubble.jl (hunk 4 above)
Comment (akshaysridhar) [reply test/test_cases/run_3d_rising_bubble.jl]: Noted, now uses Utils and updates fn name.

**Comment by akshaysridhar** [comment]
bors try

**Comment by akshaysridhar** [comment]
bors try

**Pull request merged**